packages/backend/src/services/RenameService/RenameService.ts

```diff
@@ -207,15 +207,15 @@ export class RenameService extends BaseService {
                 ];
 
                 if (fieldsInChart.includes(to)) {
-                    // It could be a table calculation / dimension / custom metric
+                    // The target field is a custom field (table calculation, custom metric, or custom dimension)
                     this.logger.info(
-                        `Replacing field "${to}" with "${from}" from the chart "${chart.name}"`,
+                        `Replacing field "${from}" with "${to}" from the chart "${chart.name}"`,
                     );
                     throw new NotImplementedError(
-                        `Replacing fields on custom chart fields is not currently supported`,
+                        `Renaming to a custom field (table calculation, custom metric, or custom dimension) is not currently supported. You can rename FROM a custom field, but not TO one.`,
                     );
                 } else {
-                    // Otherwise, it will keep failing
+                    // The target field should be a standard field from the explore
```

Comment:
updated docs around this, I found the previous a bit confusing, so let me know if this makes sense 🙏